content/sensu-go/6.7/observability-pipeline/observe-schedule/checks.md

```diff
@@ -560,6 +560,120 @@ Learn how to use check tokens with the [Sensu tokens reference documentation][5]
 **NOTE**: Check tokens are processed before check execution, so token substitutions will not apply to check data delivered via the local agent socket input.
 {{% /notice %}}
 
+## Subdues
+
+Use the [`subdues` attribute][75] in check definitions to set specific periods of time when Sensu will not send alerts based on the events the check produces.
+Subdues allow you to schedule alert-free periods of time, such as during sleeping hours, weekends, or special maintenance periods.
```

Comment:
I'm not sure if this is what subdues do. It seems like subdued checks still produce events, but not alerts. I don't know if that's what we mean by subdue. Does it mean something else, like the check doesn't execute at all or the check executes but Sensu doesn't record the event?

Reply:
Subdued checks will simply not execute at all during the specified period.